keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -146,10 +143,170 @@ _This section must be completed when targeting alpha to a release._
     - Feature gate name: PreferNominatedNode
     - Components depending on the feature gate: kube-scheduler
 
+* **Does enabling the feature change any default behavior?**
+  Yes. The pod with the nominated node has been set will be evaluated first in any scheduling cycle,
```

Comment:
Sorry - I think I'm still a bit lost.

In the description above you wrote:
```
In filtering phase, which is currently implemented in the method of `findNodesThatFitPod`, check the nominated node first if the incoming pod has the `.status.nominatedNodeName` defined and the feature gate is enabled.	 
```

That isn't consistent with what you write here that you consider *the pod with nominatedNode first".

What exactly you're going to do?

Reply:
Sorry for confusion, they are talking the same thing, 
"The pod with the nominated node has been set will be evaluated first" is actually evaluated in the [filtering, scoring], `findNodesThatFitPod` is the core method that implements for filtering,  since only one node (the nominated node ) will return if this node passes all the filtering, the scoring phase will be skipped in the current code.

Reply:
It's not the pod that will be evaluated first - it's its nominated node.

Reply:
So I'm suggesting to switching to:

"For pods with nominated node set, their nominated node will be evaluated first in any scheduling cycle..."

Reply:
Done